Watchman.Web/Areas/JwtAuth/Infrastructure/JwtRefreshTokenCache.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.Extensions.Hosting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Watchman.Web.Areas.JwtAuth.Infrastructure
+{
+    public class JwtRefreshTokenCache : IHostedService, IDisposable
+    {
+        private Timer _timer;
+        private readonly IJwtAuthManager _jwtAuthManager;
+
+        public JwtRefreshTokenCache(IJwtAuthManager jwtAuthManager)
+        {
+            _jwtAuthManager = jwtAuthManager;
```

Comment:
przed każdym polem this. (tak u nas w projekcie przyjęliśmy po prostu, że robimy pola jako this._nazwa)

Reply:
ok zmieniam.